### PR TITLE
Update to the latest version of Ruru

### PR DESCRIPTION
**Pull request by d-unseductable**
I've spent some time this week working on improving API of Ruru and on the build/usage process.

 - Use `class!` and `methods!` macros from Ruru `0.6.0`
 - Remove `Fiddle` and require the `.so` file directly to Ruby

### CODE REVIEW

File: ruru/src/lib.rs [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
 
-    let mut hash = Hash::new();
+methods!(
+    Calculator,
+    itself,
 
-    for i in 1..num + 1 {
-        hash.store(Fixnum::new(i), Fixnum::new(i.pow(3)));
-    }
+    pow_3(num: Fixnum) -> Hash {
```
Comment (hoodie) [inline ruru/src/lib.rs]: hey, what speaks against the `fn` keyword in this macro? It would retain the original Rust syntax

Comment (d-unseductable) [inline ruru/src/lib.rs]: Hey @hoodie! Thanks, that's really cool idea! I will change the macro to use `fn`

@steveklabnik please do not merge the PR, I will update it tomorrow as well.

**Comment by d-unseductable** [comment]
@steveklabnik @hoodie the example was updated to use `fn` keyword

```rust
fn pow_3(num: Fixnum) -> Hash {
    let mut hash = Hash::new();

    for i in 1..num.to_i64() + 1 {
        hash.store(Fixnum::new(i), Fixnum::new(i.pow(3)));
    }

    hash
}
```

**Comment by steveklabnik** [comment]
Awesome, thanks!

**Pull request merged**